### PR TITLE
feat(RadioButton): redesign to increase contrast in component

### LINKED ISSUE #1732: More contrast look of RadioButton
Internal ticket - UXRFC-314

### PR DESCRIPTION
**Pull request by benax-se**
Closes #1732

### CODE REVIEW

File: src/components/RadioButton/RadioButton.scss [hunk 1]
```diff
@@ -3,45 +3,94 @@
...
+            background-color: var(--g-color-line-generic);
+        }
+
+        &::after {
             content: '';
             position: absolute;
-            z-index: -1;
-            inset: 3px;
-            border-radius: var(--_--border-radius-inner);
+            z-index: -2;
```
Comment (zamkovskaya) [inline src/components/RadioButton/RadioButton.scss:39]: In which case is z-index: -1 not sufficient?

**Review by zamkovskaya — APPROVED** [review]

File: src/components/RadioButton/RadioButton.scss (hunk 1 above)
Comment (benax-se) [reply src/components/RadioButton/RadioButton.scss]: I forgot to lower it to -1 after testing different layout options.
Fixed it

**Comment by amje** [comment]
This PR might be fixing the issue #1923

**Comment by ogonkov** [comment]
@amje looks like this PR could soulve this problem, yes, is there eta when this PR could be merged?

File: src/components/RadioButton/RadioButton.scss [hunk 2]
```diff
@@ -3,45 +3,94 @@
...
         text-align: center;
-        border-radius: var(--_--border-radius-inner);
+
         cursor: pointer;
         transform: scale(1);
-        transition: color 0.15s linear;
+        transition: color var(--_--transition-time) linear;
 
-        &-outline {
+        &::before {
```
Comment (ogonkov) [inline src/components/RadioButton/RadioButton.scss:27]: What is the reason to use pseudo-elements? Why just don't change border and of element itself?

**Pull request merged**